### ISSUE TITLE
env vars not parsed correctly

### ISSUE THREAD

**Issue by xmonader** [threefoldtecharchive/zosctl#51]
parsing of environment variables passed to container new breaks in case of url (anything with colon)

**Comment by xmonader** [comment]
fixed here fb289df4e688a0c502413457b1a2e3908541b1b0

**Comment by Dina-Abd-Elrahman** [comment]
# ZOS Version:
23d648312221ff8e4d0d938e22d7d5aacd19b70e

# Verification Steps:
- zos container new --name=testssh2 --root="https://hub.grid.tf/thabet/redis.flist" --env="test:https://github.com"

```
{
  "cpu": 0,
  "rss": 5763072,
  "vms": 421249024,
  "swap": 0,
  "container": {
    "arguments": {
      "root": "https://hub.grid.tf/thabet/redis.flist",
      "mount": null,
      "host_network": false,
      "identity": "",
      "nics": [
        {
          "type": "default",
          "id": "",
          "hwaddr": "",
          "config": {
            "dhcp": false,
            "cidr": "",
            "gateway": "",
            "dns": null
          },
          "monitor": false,
          "state": "configured"
        },
        {
          "type": "zerotier",
          "id": "9bee8941b5717835",
          "hwaddr": "",
          "config": {
            "dhcp": false,
            "cidr": "",
            "gateway": "",
            "dns": null
          },
          "monitor": false,
          "state": "configured"
        }
      ],
      "port": {},
      "privileged": false,
      "hostname": "testssh2",
      "storage": "zdb://hub.grid.tf:9900",
      "name": "testssh2",
      "tags": null,
      "env": {
        "test": "https://github.com"
      },
      "cgroups": [
        [
          "devices",
          "corex"
        ]
      ],
      "config": {
        "/root/.ssh/authorized_keys": "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDHm7tHFg8+I6fPjyTOzMHF/MvZ7p8xyiDFIogFqcuIBoiyMSFaxPrhXZg4E3z73eGZ+o/axQwbdPtpszjzCjgtXvzLAaEdr4cKn9r78WUQP9egH/QS4XxA+qf8a8kQA4VojP7pb5G5P0HNfX5IgM3gRHJPq8pGBYgstI7T+q+bGsDvn29Y0AFMUbtKS6R24L9EZ+Ns5L/uWjWkIXh9Fab93rtGWoygscKIoWEfi8VHYjvl1JIuWHL6bTePTL5Q7co5ospGKEaiIysSM1vhol8uv1Mphp2oarQNdYhcva4IFsbPw89x7FIdOlzYkFZp/MKKEuHGK5w1tvUhkxtc9yxl root@dina-Inspiron-5423\n"
      }
    },
    "root": "/mnt/containers/161",
    "pid": 28863
  }
}
```
